### ISSUE TITLE
backporting bot

### ISSUE THREAD

**Issue by jdkent** [poldracklab/fmriprep#1956]
I just came across [Meeseeksdev in pandas](https://github.com/pandas-dev/pandas/pull/31183#issuecomment-580712333) and i wanted to make a note since it could be useful since fmriprep is getting into the backporting business.

**Comment by effigies** [comment]
Yeah, that seems fine with me. Hopefully we're not doing that much backporting. The goal is really to just have bug-fix releases that aren't tied to new features that are likely introducing new and exciting bugs or refactoring things so that working directories become outdated.
